docs/Custom-Brimcap-Config.md

```diff
@@ -120,8 +120,10 @@ wrapper scripts are copied to `/usr/local/bin`, the configuration can be tested
 outside the app to import a `sample.pcap` like so:
 
 ```
-$ /opt/Brim/resources/app.asar.unpacked/zdeps/zed api create -p testpool
-$ /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -root "$HOME/.config/Brim/data/brimcap-root" -config zeek-suricata.yml -p testpool sample.pcap
+$ export ZDEPS="/opt/Brim/resources/app.asar.unpacked/zdeps"
+$ $ZDEPS/zed api create -p testpool
+$ $ZDEPS/brimcap analyze -config zeek-suricata.yml sample.pcap | $ZDEPS/zed api load -p testpool -
+$ $ZDEPS/brimcap index -root "$HOME/.config/Brim/data/brimcap-root" -r sample.pcap
```

Comment:
Thanks @nwt. Indeed, your PR feedback in the Brim PR brought me around to setting `PATH` rather than that `$ZDEPS/` approach I was using, so I've pushed a commit to do the same here.

Regarding the other suggestions, we might have to agree to disagree on those. Regarding quotes around variables, while I know it's not often _necessary_, I grow weary of `shellcheck` constantly reminding me when I forget to do it, so I've conditioned myself to quote whenever I'm expanding. Likewise, I sometimes forget when some child process is actually going to need one of these variables, so I sometimes use `export` when it's not strictly necessary. Feel free to contact me directly if you feel I must cease & desist from this behavior. :smile: